### PR TITLE
[4.0] Bump Node and NPM versions

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue # .

- Node > 8.11.4
- NPM > 6.4.0

### CODE REVIEW

**Review by dgrammatiko — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
     "url": "https://github.com/joomla/joomla-cms.git"
   },
   "engines": {
-    "node": ">=8.9.4",
-    "npm": ">=5.6.0"
+    "node": ">=8.11.4",
+    "npm": ">=6.4.0"
```
Comment (wilsonge) [inline package.json:12]: Is there a real need to bump the npm version? Node 8.11.4 still only ships with npm 5.6 - so no issues with the node bump - but unsure about the npm bump (although being able to use npm ci is useful npm/node version: https://nodejs.org/en/download/releases/)

Comment (wojsmol) [reply package.json:12]: @wilsonge Please see https://nodejs.org/en/blog/vulnerability/august-2018-security-releases/

Comment (wojsmol) [reply package.json:12]: and npm ci is one of proposed solutions for package-lock.json changes n https://github.com/joomla/joomla-cms/pull/21772

**Comment by wilsonge** [comment]
I've commited the node version bump with 6f95f8eeff - I'm not going to merge the npm major bump until node 10 becomes lts and we can upgrade both node and npm version's at the same time (as npm 5.6 is what ships with all node 8 versions)

**Pull request closed without merging**